tests/test_csc.py

```diff
@@ -530,9 +556,10 @@ async def test_configure_velocity(self):
             await self.remote.cmd_configureVelocity.set_start(
                 vlimit=new_limit, timeout=STD_TIMEOUT
             )
-            data = await self.remote.evt_configuration.next(
-                flush=False, timeout=STD_TIMEOUT
-            )
+
+            await asyncio.sleep(1.0)
```

Comment:
same as before, flush before sending the command and use `next` as in the original case.

Reply:
Fixed in the commit: [f03b120](https://github.com/lsst-ts/ts_mtrotator/pull/82/commits/f03b120f7da830090949ea0b1a365e87fd3689b1). Thanks!